### ISSUE TITLE
NEMA should support cross phase switching

### ISSUE THREAD

**Issue by mschrader15** [eclipse/sumo#9983]
_Cross phase switching_ in a NEMA controller is when detector calls are mapped to a different phase **when the map-to phase is active**. A basic example is mapping the [1, 5] left turns to [2, 6] when [2, 6] is active.

I have two ideas for the configuration file. @qichaow @namdre what are your thoughts?

```
<param key="crossPhaseSwitching" value="{2: 5, 6: 1}">
```

```
  <param key="crossPhaseSwitching">
      <map detector="1" activePhase="6"/>
      <map detector="5" activePhase="2"/>
  </param key="crossPhaseSwitching">
```

**Comment by namdre** [comment]
I would prefer the following as it's more in line with existing params link linkMinDur:x (https://sumo.dlr.de/docs/Simulation/Traffic_Lights.html#parameters)

```
<param key="crossPhaseSwitching:2" value="5"/>
<param key="crossPhaseSwitching:6" value="1"/>
```

**Comment by mschrader15** [comment]
That works for me.

**Comment by mschrader15** [comment]
I need to add this to documentation

**Comment by mschrader15** [comment]
Closed with #10598
